.circleci/config.yml

```diff
@@ -0,0 +1,54 @@
+version: 2.1
```

Comment:
Is this project not currently being deployed via CircleCI? Is that why this file is a new file?

Reply:
@kazuuchikata this is a gem that being used to run most of the state machine logic inside of snapsheet-tx. This circleCI will only run the test, we didn't have it running previously. 
There are a ton of test of the state machine inside of snapsheet-tx. The new version of the gem here, will need to also be updated in the snapsheet-tx repo, and those test will have to pass. I will probably update the Statesman gem inside Snapsheet-tx as part of different PRs to make sure we can fully test everything. Right now it's throwing deprecations and the one code fix is necessary for the rest of the plan.

Reply:
Ah that makes me feel better about CircleCI. Thanks for providing the context, and you're right these changes in and of themselves shouldn't do anything. Thanks!